testers/testers/sql/bin/install.sh

```diff
@@ -7,16 +7,30 @@ install_packages() {
     sudo apt-get install python3 postgresql
 }
 
+random_pwd() {
+    date +%s%N | sha256sum | base64 | head -c 32
+}
+
 create_db_and_users() {
     echo "[SQL-INSTALL] Creating databases and users"
     echo "[SQL-INSTALL] Creating oracle user '${ORACLEUSER}' with database '${ORACLEDB}'"
     sudo -u postgres psql <<-EOF
 		DROP DATABASE IF EXISTS ${ORACLEDB};
 		DROP ROLE IF EXISTS ${ORACLEUSER};
 	EOF
-    echo "[SQL-INSTALL] Create password for oracle user '${ORACLEUSER}'"
-    sudo -u postgres createuser -P ${ORACLEUSER} # secure, password is not logged
+
+    if id -u ${ORACLEUSER} &> /dev/null; then
+        sudo -u "${ORACLEUSER}" -- bash -c "touch ${HOME}/.pgpass && chmod 600 ${HOME}/.pgpass"
+        read -s -p "[SQL-INSTALL] Create password for oracle user '${ORACLEUSER}': " ORACLEPWD
+        echo ''
+        sudo -u "${ORACLEUSER}" -- bash -c "builtin echo '*:*:*:${ORACLEUSER}:${ORACLEPWD}' > ${HOME}/.pgpass"
```

Comment:
Should be as restrictive as possible (or did you have problems with it?):
`localhost:5732:${ORACLEUSER}:${ORACLEUSER}:${ORACLEPWD}`

I'm a bit concerned by the pwd being logged in the bash history, but I guess this installation is useful to try things rather than deploying in production.